deploy/charts/cert-manager/Chart.template.yaml

```diff
@@ -3,6 +3,7 @@ name: cert-manager
 # The version and appVersion fields are set automatically by the release tool
 version: v0.1.0
 appVersion: v0.1.0
+kubeVersion: ">= 1.16.0-0"
```

Comment:
Note: The `-0` suffix is required for images with additional metadata as a suffix. This is common on EKS/GCP/etc, KIND, and beta/alpha versions of Kubernetes.

For example, without it: `1.16.0-EKS` would not satisfy the condition `>= 1.16.0`

Reply:
> Note: The `-0` suffix is required for images with additional metadata as a suffix. This is common on EKS/GCP/etc, KIND, and beta/alpha versions of Kubernetes.
> For example, without it: `1.16.0-EKS` would not satisfy the condition `>= 1.16.0`

I have some doubts. 
I would have expected `kubeVersion: ">= 1.16"` to work.

* I think Helm gets K8S version by querying the API server `/version` endpoint.
* Are you saying that the K8S API server on EKS always return a version which has the `-EKS` suffix? 
* You say that KIND and GCP also do this, but I've never observed it.

I'm happy to be convinced,  but please post some output showing how you've verified this behaviour on EKS / GCP etc.
Also please see if there are any examples of other Helm charts taking this approach and share them here.
And post any links from the Helm issue tracker which mention this problem.

I found this paragraph in https://github.com/Masterminds/semver#working-with-prerelease-versions

> SemVer comparisons using constraints without a prerelease comparator will skip prerelease versions. For example, >=1.2.3 will skip prereleases when looking at a list of releases while >=1.2.3-0 will evaluate and find prereleases.
> 
> The reason for the 0 as a pre-release version in the example comparison is because pre-releases can only contain ASCII alphanumerics and hyphens (along with . separators), per the spec. Sorting happens in ASCII sort order, again per the spec. The lowest character is a 0 in ASCII sort order (see an ASCII Table)
> 
> Understanding ASCII sort ordering is important because A-Z comes before a-z. That means >=1.2.3-BETA will return 1.2.3-alpha. What you might expect from case sensitivity doesn't apply here. This is due to ASCII sort ordering which is what the spec specifies.

But that seems to be aimed at people who want to override the default skipping of pre-releases.

I suppose if we wanted to be really precise, we should use `kubeVersion: "^1.6"` since according to those Mastermind docs, would restrict the version to `>=1.16 AND <2.0.0-0`. 
Although I doubt if there's going to be a Kubernetes v2 in the lifetime of this cert-manager release :-)